src/framework/uicomponents/qml/MuseScore/UiComponents/Utils.js

```diff
@@ -60,3 +60,12 @@ function getItem(model, row) {
 
     return model[row]
 }
+
+function makeMnemonicText(text){
```

Comment:
Missing space between `)` and `{`, I wonder why the codestyle check didn't catch that?

Reply:
thanks, fixed